src/main/java/svc/models/Municipality.java

```diff
@@ -0,0 +1,13 @@
+
+package svc.models;
+
+import java.util.List;
+
+
+public class Municipality {
```

Comment:
Why not an `@Entity` ?

Reply:
I deleted this file, because this implementation is supposed to deal only with Court and Judge